en/simulation/README.md

```diff
@@ -159,32 +159,64 @@ The simulated camera is a gazebo plugin that implements the [MAVLink Camera Prot
+This does not work "out of the box" because PX4 does not route packets to external interfaces by default (in order to avoid spamming the network and different simulations interfering with each other). Instead it routes traffic internally - to "localhost".
 
-One, probably the easiest way to create the tunnel is the use of SSH tunneling options.  The tunnel itself could be created easily by running the following command on localhost.
+There are a number of ways to make the UDP packets available on external interfaces, as outlined below.
+ 
+
+### Enable MAV_BROADCAST
+
+Enable [MAV_BROADCAST](../advanced/parameter_reference.md#MAV_BROADCAST) to broadcast heartbeats on the local network.
+
+A remote computer can then connect to it by listening to the Simulation computer's IP address on the appropriate port (i.e. 14550 for *QGroundControl*).
+
+
+### Use MAVLink Router
+
+[mavlink-router](https://github.com/intel/mavlink-router) can be used to route  packets from localhost to an external interface.
+
+### Modify Configuration for External Broadcasting 
+
+The [mavlink](../middleware/modules_communication.md#mavlink_usage) module routes to *localhost* by default, but you can specify an external IP address to broadcast to using its `-t` option.
+
+This should be done in various configuration files where `mavlink start` is called. For example: [/ROMFS/px4fmu_common/init.d-posix/rcS](https://github.com/PX4/Firmware/blob/master/ROMFS/px4fmu_common/init.d-posix/rcS).
+
+
+### SSH Tunneling
+
+SSH tunneling is a flexible option because the simulation computer and the system using it need not be on the same network.
+
+> **Note** You might similarly use VPN to provide a tunnel to an external interface (on the same network or another network).
+
+One way to create the tunnel is to use SSH tunneling options.
+The tunnel itself can be created by running the following command on *localhost*, where `remote.local` is the name of a remote computer:
 ```
 ssh -C -fR 14551:localhost:14551 remote.local
-```
-Where "remote.local" is the name of a remote computer. 
+``` 
 
-Unfortunately, the SSH itself cannot route the UDP packets.  Therefore the UDP packets need to be translated to TCP packets. By [netcat](https://en.wikipedia.org/wiki/Netcat) utility separately on the local and remote side of the tunnel. 
-Local side of UDP packet translation of QGC could be implemented by running following commands.
+The UDP packets need to be translated to TCP packets so they can be routed over SSH. 
+The [netcat](https://en.wikipedia.org/wiki/Netcat) utility can be used on both sides of the tunnel - first to convert packets from UDP to TCP, and then back to UDP at the other end.
+
+> **Tip** QGC must be running before executing *netcat*.
+ 
+On the *QGroundControl* computer, UDP packet translation may be implemented by running following commands:
```

Comment:
I reworded to refer to QGC side of tunnel and SITL side - please check I got it right.

Reply:
Looks correct.

Reply:
Yes, it is correct.